README.md

```diff
@@ -634,31 +634,27 @@ When running a binary (`pipx run BINARY`), pipx will
 These are all things you can do yourself, but pipx automates them for you. If you are curious as to what pipx is doing behind the scenes, you can always pass the `--verbose` flag to see every single command and argument being run.
 
 ## Contributing
-To develop `pipx` first clone the repository, then create and activate a virtual environment.
+To develop `pipx` first clone the repository, have tox with tox-venv installed somewhere, then run tox.
 ```
-python3 -m venv venv
-source venv/bin/activate
+tox --notest
```

Comment:
I ran this from inside a venv, and it worked. I then activated the tox venv with `source .tox/python/bin/activate` which worked. I then ran `tox` but I got an error: 
```
>> tox

Command 'tox' not found, but can be installed with:

sudo apt install tox
```

I then deactivated the venv, ran `python3 -m pip install --user tox tox-venv`, then `tox --notest`, then `tox`, and now the tests are running. 

Do tox and tox-venv need to be installed outside of a venv? If so, can this be added to the docs as well?

Reply:
`tox` is just like any other Python console script program. If you install it in a venv, you have to run it from that venv or have that venv activated to run tox. If you `activate` the venv that tox creates in order for `pipx` to be on the path, then `tox` will no longer be on the path. Tox creates multiple venvs and thus must necessarily be outside of the venvs it creates. I personally almost never "activate" venvs, because it's a platform-sensitive operation. I'll update the docs to reflect that recommendation, which I believe is nicer for tox-based workflows.